content/6_Advanced/Multiplicative.mdx

```diff
@@ -1,24 +1,227 @@
 ---
 id: multiplicative
 title: 'Prefix Sums of Multiplicative Functions'
-author: Benjamin Qi
-description: '?'
+author: Tianqin Meng,  Benjamin Qi
+description: The module covers prefix sums for multiplicative functions, defining arithmetic, multiplicative, and completely multiplicative functions. It explains Dirichlet convolution, Möbius inversion, and binomial inversion, with examples like summing divisors, totient calculations, enumerating primes, and counting primes, along with detailed explanations and implementations.
```

Comment:
```suggestion
description: Covering Dirichlet convolution, Möbius inversion, and binomial inversion.
```
no need to be too specific lol (was this chatgpt 😭)

Reply:
ye  🙏 ill tell him to cut it down abit (mkade him cook coz kevin told me it was too short before😭 )

Reply:
wasn't that it was too short, he said it wasn't informative enough (before it was just a copy of the title)

Reply:
okay understood